src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str.cs

```diff
@@ -228,4 +223,44 @@ public void UnixValidCharsDirectoryPath()
+            Directory.CreateDirectory(Path.Combine(testDir, GetTestFileName()));
+            RemoteInvoke((testDirectory) =>
+            {
+                Directory.SetCurrentDirectory(testDirectory);
+                Assert.NotEmpty(Directory.GetFileSystemEntries(Directory.GetCurrentDirectory()));
+                Assert.NotEmpty(Directory.GetFileSystemEntries(Directory.GetCurrentDirectory(), "*"));
+                Assert.NotEmpty(Directory.GetFileSystemEntries(Directory.GetCurrentDirectory(), "*", SearchOption.AllDirectories));
+                Assert.NotEmpty(Directory.GetFileSystemEntries(Directory.GetCurrentDirectory(), "*", SearchOption.TopDirectoryOnly));
+                Assert.NotEmpty(Directory.GetDirectories(Directory.GetCurrentDirectory()));
+                Assert.NotEmpty(Directory.GetDirectories(Directory.GetCurrentDirectory(), "*"));
+                Assert.NotEmpty(Directory.GetDirectories(Directory.GetCurrentDirectory(), "*", SearchOption.AllDirectories));
+                Assert.NotEmpty(Directory.GetDirectories(Directory.GetCurrentDirectory(), "*", SearchOption.TopDirectoryOnly));
+                Assert.NotEmpty(Directory.GetFiles(Directory.GetCurrentDirectory()));
+                Assert.NotEmpty(Directory.GetFiles(Directory.GetCurrentDirectory(), "*"));
+                Assert.NotEmpty(Directory.GetFiles(Directory.GetCurrentDirectory(), "*", SearchOption.AllDirectories));
+                Assert.NotEmpty(Directory.GetFiles(Directory.GetCurrentDirectory(), "*", SearchOption.TopDirectoryOnly));
+                Assert.NotEmpty(Directory.EnumerateFileSystemEntries(Directory.GetCurrentDirectory()));
+                Assert.NotEmpty(Directory.EnumerateFileSystemEntries(Directory.GetCurrentDirectory(), "*"));
+                Assert.NotEmpty(Directory.EnumerateFileSystemEntries(Directory.GetCurrentDirectory(), "*", SearchOption.AllDirectories));
+                Assert.NotEmpty(Directory.EnumerateFileSystemEntries(Directory.GetCurrentDirectory(), "*", SearchOption.TopDirectoryOnly));
+                Assert.NotEmpty(Directory.EnumerateDirectories(Directory.GetCurrentDirectory()));
+                Assert.NotEmpty(Directory.EnumerateDirectories(Directory.GetCurrentDirectory(), "*"));
+                Assert.NotEmpty(Directory.EnumerateDirectories(Directory.GetCurrentDirectory(), "*", SearchOption.AllDirectories));
+                Assert.NotEmpty(Directory.EnumerateDirectories(Directory.GetCurrentDirectory(), "*", SearchOption.TopDirectoryOnly));
+                Assert.NotEmpty(Directory.EnumerateFiles(Directory.GetCurrentDirectory()));
+                Assert.NotEmpty(Directory.EnumerateFiles(Directory.GetCurrentDirectory(), "*"));
+                Assert.NotEmpty(Directory.EnumerateFiles(Directory.GetCurrentDirectory(), "*", SearchOption.AllDirectories));
+                Assert.NotEmpty(Directory.EnumerateFiles(Directory.GetCurrentDirectory(), "*", SearchOption.TopDirectoryOnly));
+                return SuccessExitCode;
```

Comment:
Nit: consider adding a blank line between each logical section of tests above, just to help make it easier to see the groupings